Ejercicio Applicant-Tracking System

moncio opened the pull request:
Analisis y diseño de un sistema inteligente de candidatos para trabajo utilizando asistentes IA

alvarotech commented:
Buen trabajo Tomás!

Muy buena propuesta de producto, con elementos innovadores en todos los pasos del customer journey (IA,automatización, análisis predictivos...)
El modelo de datos está bien. Sin embargo, estaría bien saber qué es una Primary Key y Foreign Key. Además, me faltan entidades para representar a los usuarios internos de la aplicación, recruiters y managers, que son clave.

A partir de aquí los resultados son muy genéricos. Tanto el diseño de sistema, como la arquitectura, no da detalles de ningun tipo de tecnología ni arquitectura particular, esperamos bastante más profundidad en este aspecto.

El diagrama C4 está bien pero de nuevo un poco simple, y le falta el nivel 4, detalles a nivel código de alguna de las clases principales.

Lo principal a nivel de feedback es que no has incorporado ningún diagrama, uno de los objetivos principales de este ejercicio. Ni mermaid, ni plantUML. Esta parte tienes que explorarla más a fondo, tanto exportar los plantUML creados y adjuntarlos, como explorar mermaid, para los casos de uso por ejemplo.

Respecto al prompt, echo de menos más conversación. Con tantos apartados lo ideal hubiera sido ir paso a paso, revisando cada fase a fondo y con ese contexto seguir pidiendo.

Respecto a tu pregunta, en md ``` indica fragmento de código. Con las letras pl indica el tipo de lenguaje utilizando, pudiendo github interpretar y destacar adecuadamente los elementos de la sintaxis.

moncio committed:
Ejercicio Applicant-Tracking System

Analisis y diseño de un sistema inteligente de candidatos para trabajo utilizando asistentes IA
